Sources/Datadog/Core/Swizzling/MethodSwizzler.swift

```diff
@@ -30,11 +30,15 @@ internal class MethodSwizzler<TypedIMP, TypedBlockIMP> {
         }
     }
 
-    private var implementationCache: [FoundMethod: IMP] = [:]
+    private var implementationCache: [FoundMethod: IMP]
     var swizzledMethods: [FoundMethod] {
         return Array(implementationCache.keys)
     }
 
+    init() throws {
+        implementationCache = [:]
+    }
```

Comment:
OK for now 👍 , but this really shows that something is wrong in there - this `init()` doesn't need to throw, but all its overwrites need to throw because of calling `Self.findMethod(...)`. We may want to refactor it at some point.

Reply:
removed ✅